### PR TITLE
update README with Project Info

### PR DESCRIPTION
**Pull request by lfryc**
https://issues.jboss.org/browse/AEROGEAR-1591

### CODE REVIEW

**Comment by lfryc** [comment]
@corinnekrych @cvasilak could you please review?

I can send updates against rest of the repos once you confirm this one.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ Taking care of:
 
 100% Swift.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
```
Comment (lfryc) [inline README.md]: Noticed forgot to change `Maven` build ;-)

What would be the best way to describe the build system here?

Comment (corinnekrych) [inline README.md]: cocoapods

**Comment by corinnekrych** [comment]
+1

File: README.md (hunk 1 above)
Comment (danielpassos) [inline README.md]: :+1:  for cocoapods. Wdyt @cvasilak?

**Comment by matzew** [comment]
+1 on Cocoapods :) 

and +1 on the content

**Comment by cvasilak** [comment]
thanks @lfryc +1 on 'cocoapods'

**Comment by lfryc** [comment]
Updated the content, @cvasilak could you please review once more and merge?

**Comment by cvasilak** [comment]
:+1:  thanks @lfryc landed

**Pull request merged**